testament/testament.nim

```diff
@@ -487,7 +489,7 @@ proc testSpecHelper(r: var TResults, test: var TTest, expected: TSpec,
             args = @["--unhandled-rejections=strict", exeFile] & args
           else:
             exeCmd = exeFile.dup(normalizeExe)
-            if expected.useValgrind != disabled:
+            if useValgrind and expected.useValgrind != disabled:
```

Comment:
Shouldn't this be `useValgrind or ...`?

Reply:
`useValgrind` is a global variable. Probably should be renamed to avoid confusion